lib/browser.ts

```diff
@@ -10,6 +10,11 @@
  * @mergeTarget
  */
 
+import * as iotidentity from './iotidentity/iotidentityclient';
+import * as greengrass from './greengrass/discoveryclient';
+import * as iotjobs from './iotjobs/iotjobsclient';
+import * as iotshadow from './iotshadow/iotshadowclient';
```

Comment:
These references break the actual browser compatibility, as they pull in native components.